data/org.freedesktop.portal.Clipboard.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0"?>
+<!--
+ Copyright 2022 Google LLC
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<node name="/"
+  xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+   org.freedesktop.portal.Clipboard
+   @short_description: Clipboard portal
+
+   The Clipboard portal allows clipboard control over remote desktop sessions.
+
+   This documentation describes version 1 of this interface.
+  -->
+  <interface name="org.freedesktop.portal.Clipboard">
+    <!-- 
+    EnableClipboard:
+    @session_handle: Object path for the #org.freedesktop.portal.Session object
+    @options: Options for the clipboard
+    @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+    Enables clipboard control for the given remote desktop session.
+
+    Supported keys in the @options vardict include:
+    <variablelist>
+      <varlistentry>
+        <term>handle_token s</term>
+        <listitem><para>
+          A string that will be used as the last element of the @handle. Must be a valid
+          object path element. See the #org.freedesktop.portal.Request documentation for
+          more information about the @handle.
+        </para></listitem>
+      </varlistentry>
+    </variablelist>
+    -->
+    <method name="EnableClipboard">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+    <!-- 
+      SetSelection:
+      @session_handle: Object path for the #org.freedesktop.portal.Session object
+      @options: Vardict with optional further information
+      @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
```

Comment:
Missing documentation for what the method actually does. This applies to the other methods that follows this one as well.